_storage/write-to-read.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Storage write and read
+
+Once a user authenticates and a DApp obtains authentication, the application interacts with Gai through the blockstack.js library. There are two simple methods for working with data in Gaia hub the `putFile()` and `getFile()` methods. This section goes into greater detail about the methods, how they interact with a hub, and how to use them.
+
+
+## Write-to and Read-from URL Guarantees
+
+Gaia is built on a driver model that supports many storage services. So, with very few lines of code, you can interact with providers on Amazon S3, Drop, and so forth.  The simple getFile and putFile interfaces are kept simple because Blockstack assumes and wants to encourage a community of OS data management libraries.
```

Comment:
"storage providers" instead of "services"?

Reply:
"Dropbox" instead of "Drop"

Reply:
Should we always highlight method names with double back hashes aka `getFile` instead of getFile here?

Reply:
"open-source data-management libraries"